src/common/helpers/currentSigningMessage.ts

```diff
@@ -0,0 +1,27 @@
+import web3 from "web3";
+
+// Note: this will be used once we sign more rarely, instead of once per round.
+// ATM, we want to keep the current and old voter dapp aligned.
+export default function currentSigningMessage(currentRound: number) {
+  /*
+    Whenever we change the signing message, we should wait until the next round.
+    Determine current round, then create a conditional such that if we change the message in the middle of the round, 
+    it will only apply to the next, to prevent any bugs in revealing.
+    There shouldn't be many reasons for us to change this, this is just to prevent any bugs in the commit -> reveal in a given round.
+  */
+  if (currentRound > SIGNING_MESSAGE_ONE_ROUND_CREATED) {
+    return {
+      message: SIGNING_MESSAGE_ONE,
+      hashedMessage: web3.utils.utf8ToHex(SIGNING_MESSAGE_ONE),
+    };
+  }
+
+  // Default return
+  return {
+    message: SIGNING_MESSAGE_ONE,
+    hashedMessage: web3.utils.utf8ToHex(SIGNING_MESSAGE_ONE),
+  };
```

Comment:
these branches are returning the exact same thing, doesnt seem right

Reply:
It is correct in this case.

If you can imagine we only change the message every N period, we add a hardcoded value. We could just add in the conditional statement later once a message changes, but it's to show how the code works after a message changes.